.travis.yml

```diff
@@ -5,8 +5,9 @@ script:
 - lein clean
 - lein compile
 - lein check
-- lein eastwood '{:exclude-namespaces [hiposfer.kamal.graph.specs hiposfer.kamal.spec]}'
+- lein eastwood '{:exclude-namespaces [hiposfer.kamal.graph.specs hiposfer.kamal.spec]}'  
 - lein test
+- lein jar # produce jar files to deploy
```

Comment:
@mehdisadeghi I think this should be `lein uberjar`. That is probably why you only get `kamal*.jar` file and not `kamal.jar` as specified in our `project.clj`

Reply:
@carocad `lein jar` produces kamal-VERSION.jar and `lein uberjar` produces `kamal.jar` because it is hardcoded in the `project.clj` file. Nevertheles, with current config both will work because I have used a wildcard to select files to be added to the GH release.